src/AppInstallerCommonCore/Manifest/ManifestCommon.cpp

```diff
@@ -481,6 +481,14 @@ namespace AppInstaller::Manifest
             );
     }
 
+    bool DoesInstallerTypeIgnoreScopeFromManifest(InstallerTypeEnum installerType)
+    {
+        return (

```

Comment:
Should this also check nested installer type if a package is an archive, or does that not matter here?

Reply:
Indeed, it might be productive to make all of these semantic functions take in an `Installer` rather than a type so that we force it the check to be done properly.